storage/plugins/z39-server/classes/UNIMARCXMLFormatter.php

```
@@ -0,0 +1,198 @@
+
+        return $recordEl;
+    }
+
+    // ── DOM helpers ───────────────────────────────────────────────────────────
+
+    private function cf(string $tag, string $value): \DOMElement
+    {
+        $field = $this->doc->createElement('controlfield', $this->escapeXml($value));
+        $field->setAttribute('tag', $tag);
+        return $field;
+    }
```

PHPMD (2.15.0)

[warning] Avoid using short method names like UNIMARCXMLFormatter::cf(). The configured minimum method name length is 3. (undefined)

(ShortMethodName)